internal/pkg/manifest/workload.go

```diff
@@ -350,6 +352,7 @@ type DockerBuildArgs struct {
 	Dockerfile *string           `yaml:"dockerfile,omitempty"`
 	Args       map[string]string `yaml:"args,omitempty"`
 	Target     *string           `yaml:"target,omitempty"`
+	Platform   string            `yaml:"platform,omitempty"`
```

Comment:
Is it possible to define `Platform` in a single location? preferably in `TaskConfig` 🙏 

This will allow users to define `platform` under `image.build` because of the `yaml` tags which I think we are trying to avoid 🤔 

Maybe we can do it through 💭 
```go
func (s *BackendService) TaskPlatform() string
func (s *BackendService) BuildArgs() *DockerBuildArgs
```